src/lib/utils/locker.ts

```diff
@@ -15,7 +19,7 @@ export const withLock = async <T>(resource: string, fn: (lockId: Date) => Promis
       lockId,
     });
   } catch {
-    return;
+    return fallback;
```

Comment:
What does adding the return type here do? Is there a noticeable difference between `false` and `undefined` in our `withLock` usages above?

Reply:
I added it when I was making a couple other changes to the lock that I reverted, but I think its cleaner with better defined return values.